modules/core/shared/src/main/scala/io/circe/Decoder.scala

```diff
@@ -1477,6 +1476,29 @@ object Decoder
         )
     )
 
+  /** Deserializing `java.util.Locale` using a IETF BCP 47 string representation. */
+  object locale {
```

Comment:
I meant something more along the lines of:

```
object locale {
  object IETF_BCP_47 {
     implicit final lazy val decoder: Decoder[Locale] ...
  }
}
```

so end users will do
`import io.circe.Decoder.locale.IETF_BCP_47._`